data/human/transport missions.txt

```diff
@@ -3582,3 +3582,53 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+mission "Earth Retirement"
+	name "Retired Family to Earth"
+	description "Transport two seniors to <destination>, which is where they plan to retire. They can only promise you <payment>."
+	minor
+	source
+		attributes paradise
+	destination "Earth"
+	to offer
+		random < 15
+	passengers 2
+	
+	on offer
+		conversation
+			`As you wander through the spaceport, you are see an elderly man and woman sitting on a bench. They both seem to be talking about something that is causing them some amount of stress.`
+			choice
+				`	"Yes. Who are you two?"`
```

Comment:
```suggestion
				`	"Do you need any help?"`
```
IMO, these two responses fit more in a situation where you're being approached by the mission offerer rather than a situation where you're approaching the mission offerer (as it appears to be here).